software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcess.java

```diff
@@ -49,14 +52,34 @@
             "indicates that a reboot should be performed after the install command is run", false);
     ConfigKey<Boolean> CUSTOMIZE_REBOOT_REQUIRED = ConfigKeys.newBooleanConfigKey("customize.reboot.required",
```

Comment:
I'd include `priority=...` on each, to force a sensible order in their listing within the wizard. Not sure what guarantees we get here (probably it's just in the order returned by http://docs.oracle.com/javase/7/docs/api/java/lang/Class.html#getFields(), which says "not sorted and are not in any particular order").

I personally would have also added `@CatalogConfig` to the other powershell commands such as `PRE_INSTALL_POWERSHELL_COMMAND` etc. It's easy enough for someone to scroll down, filling in only what they want. But if they are not annotated then that config option is very hard to use from the wizard.